main.py

```diff
@@ -1,22 +1,37 @@
-from flask import Flask, request
-from flask_json import FlaskJSON
+import json
+from flask import Flask, request, make_response
 
-app = Flask(__name__)
-FlaskJSON(app)
 
-@app.route("/")
-def hello():
-    return "Hello World!"
-
-@app.route('/login', methods=['POST'])
-def login():
+def create_app():
+    app = Flask(__name__)
     
-    data = request.get_json(force=True)
+    @app.route('/login', methods=['POST'])
+    def login():
+        
+        if 'Content-Type' not in request.headers.keys() or request.headers['Content-Type'] != 'application/json':
+            http_code = 400
```

Comment:
There is no need for a separate variable for http code, inline it, please. Applies throughout.

Reply:
Done!